dags/bq_events_to_amplitude.py

```diff
@@ -0,0 +1,79 @@
+import datetime
+
+from airflow import models
+from utils.gcp import bigquery_etl_query
+from airflow.contrib.operators.bigquery_to_gcs import BigQueryToCloudStorageOperator
+from airflow.contrib.operators.gcs_to_s3 import GoogleCloudStorageToS3Operator
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+
+default_args = {
+    'owner': 'frank@mozilla.com',
+    'start_date': datetime.datetime(2019, 6, 27),
+    'email': ['telemetry-alerts@mozilla.com', 'frank@mozilla.com'],
+    'email_on_failure': True,
+    'email_on_retry': True,
+    'depends_on_past': False,
+    'retries': 2,
+    'retry_delay': datetime.timedelta(minutes=10),
+}
+
+environment = '{{ task.__class__.deploy_environment }}'
+
+dag_name = 'bq_events_to_amplitude'
+
+# This connection needs access to both BQ and GCS
+gcp_conn_id = 'google_cloud_derived_datasets'
+
+amplitude_s3_conn = 'amplitude_s3_conn'
+amplitude_s3_bucket = 'com-amplitude-vacuum-mozilla-vacuum-wup'
+
+with models.DAG(
+        dag_name,
+        schedule_interval='0 1 * * *',
+        default_args=default_args) as dag:
+
+    # TODO update view
+    # For now, we assume the view is already updated
+    # See https://github.com/mozilla/bigquery-etl/issues/218
+
+    # Overwrite the table with yesterday's data
+    table_name = 'fenix_events_amplitude_v1'
+    table_creation = bigquery_etl_query(
+        destination_table=table_name,
+        date_partition_parameter=None,
+        parameters=('submission_date' + ':DATE:{{ds}}',)
+    )
+
+    project_id = GoogleCloudBaseHook(gcp_conn_id=gcp_conn_id).project_id
+
+    gcs_bucket = 'moz-fx-data-amplitude-export'
+    directory = '{env}/fenix/{{ ds_nodash }}/'.format(env=environment)
+    extension = '.tsv.gz'
+
+    # Export from bq to gcs
+    # Docs: https://github.com/apache/airflow/blob/master/airflow/contrib/operators/bigquery_to_gcs.py#L28
```

Comment:
wtmo runs tag 1.10.2, does master branch match for this file?

Reply:
Yes, I tested this out on dev with 1.10.2. There are some changes for 1.10.4, but nothing that effects us.

Reply:
this comment will become outdated though

Reply:
IMO better than no doc link, it will be close to where the class is defined.